release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1305,6 +1305,36 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* If the installation program is unable to get all of the projects that are associated with the Google Cloud Platform (GCP) service account, the installation fails with a `context deadline exceeded` error message.
++
+This behavior occurs when the following conditions are met:
+
+** The service account has access to an excessive number of projects.
+** The installation program is run with one of the following commands:
+*** `openshift-install create install-config`.
++
+.Error message
+[source,text]
+----
+FATAL failed to fetch Install Config: failed to fetch dependency of "Install Config": failed to fetch dependency of "Base Domain": failed to generate asset "Platform": failed to get projects: context deadline exceeded
+----
+*** `openshift-install create cluster` without an existing installation configuration file (`install-config.yaml`).
++
+.Error message
+[source,text]
+----
+FATAL failed to fetch Metadata: failed to fetch dependency of "Metadata": failed to fetch dependency of "Cluster ID": failed to fetch dependency of "Install Config": failed to fetch dependency of "Base Domain": failed to generate asset "Platform": failed to get projects: context deadline exceeded
+----
+*** `openshift-install create manifests` with or without an existing installation configuration file.
++
+.Error message
+[source,text]
+----
+ERROR failed to fetch Master Machines: failed to load asset "Install Config": failed to create install config: platform.gcp.project: Internal error: context deadline exceeded
+----
++
+As a workaround, if you have an existing installation configuration file, update it with a specific project id to use (`platform.gcp.projectID`). Otherwise, manually create an installation configuration file, and enter a specific project id. Run the installation program again, specifying the existing file. (link:https://issues.redhat.com/browse/OCPBUGS-15238[*OCPBUGS-15238*])
```

Comment:
```suggestion
As a workaround, if you have an installation configuration file, update it with a specific project id to use (`platform.gcp.projectID`). Otherwise, manually create an installation configuration file, and enter a specific project id. Run the installation program again, specifying the file. (link:https://issues.redhat.com/browse/OCPBUGS-15238[*OCPBUGS-15238*])
```
In most cases, you can remove "existing" and the sentence will retain its meaning.

Reply:
@lahinson Appreciate the review. All made good sense.